### DISCUSSION TITLE
More informative "Serial Numbers" field description

### DISCUSSION THREAD

**Discussion by ccwrapper** [inventree/InvenTree#6325] [Ideas]
New Stock Item - Serial Numbers
Currently (0.13) this field has this description: _Enter serial numbers for new stock (or leave blank)_
I would like to see more informative description, that you can enter several comma separated numbers, and more, as it [stated in the docs](https://docs.inventree.org/en/stable/part/trackable/#stock-tracking).

**Comment by SchrodingersGat** [comment]
@ccwrapper I agree that this feature could be more obvious. However it may be too much information for display in every place where serial numbers can be entered. Maybe an "extra info" icon which links to the docs?

**Reply by ccwrapper** [reply]
Maybe. I kept in mind a description like for the Choices field in Create Part Parameter Template: _Valid choices for this parameter (comma-separated)_. This already gives user an idea what is expected in this field.
